crl-janitor/variables.tf

```diff
@@ -16,6 +16,10 @@ variable "google_project" {
   type        = string
   description = "The google project in which to create resources"
 }
+variable "google_folder" {
```

Comment:
Maybe a more explicit `google_folder_id`, and also mention it in the description? Same for the terra-env variable.